pkgs/top-level/python-packages.nix

```diff
@@ -15335,6 +15335,8 @@ self: super: with self; {
 
   stamina = callPackage ../development/python-modules/stamina { };
 
+  standard-mailcap = callPackage ../development/python-modules/standard-mailcap { };
```

Comment:
```suggestion
  standard-mailcap = if pythonOlder "3.13" then null else callPackage ../development/python-modules/standard-mailcap { };
```

Reply:
I have `disabled = pythonOlder "3.13";` in the call to `buildPythonPackage`. Which one is preferred or should I put both?

Reply:
Both is good.